cmd/flags/parse.go

```diff
@@ -22,3 +23,11 @@ var FlagInputDataFileName = &cli.StringFlag{
 	Usage: "Name of file to use as raw input data",
 	Value: "raw.json",
 }
+
+const ArgOutputDirectory string = "out"
+
+var FlagOutputDirectory = &cli.StringFlag{
+	Name:  ArgOutputDirectory,
+	Usage: "Specifies an output directory to write parsed data to. Will clear directory and write fresh.",
+	Value: "./web/data",
+}
```

Comment:
Decided to finally add this because I was tired of manually moving the parsed data into the correct location.